DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: dust
 Title: Iterate Multiple Realisations of Stochastic Models
-Version: 0.7.6
+Version: 0.7.7
```

Comment:
```suggestion
Version: 0.7.8
```

Reply:
You managed to catch this just before I fixed that (we had to quickly sneak a fix in for a broken density function)